lib/src/constants/sheba/constants.dart

```diff
@@ -0,0 +1,319 @@
+var _banksInfo = <Map<String, dynamic>>[
```

Comment:
use `const `or `final` keyword instead of `var`.

Reply:
we can't use `const` because in the `call` method we should add two keys to the `Map`s but `final` is better, I will check it.